File: php/recv.php
<?php
require_once __DIR__ . '/bootstrap.php';

try {
    $client = createClient();

    $result = $client->receiveMessage([
        'QueueUrl' => QUEUE_URL,
        'MaxNumberOfMessages' => 10,
    ]);

    $messages = $result->search('Messages[]') ?? [];
    foreach ($messages as $message) {
        $handle = $message['ReceiptHandle'];
        $body = $message['Body'];
        echo $body . PHP_EOL;
        $client->deleteMessage([
            'QueueUrl' => QUEUE_URL,
            'ReceiptHandle' => $handle,
        ]);
    }

} catch (\Aws\Sqs\Exception\SqsException $e) {
    var_export($e->getMessage());
}

File: ruby/recv.rb
require './bootstrap.rb'

begin

  sqs = createClient

  result = sqs.receive_message({
    queue_url: ENV['QUEUE_URL'],
    max_number_of_messages: 10
  })

  result.messages.each do |message|
    puts message.body

    sqs.delete_message({
      queue_url: ENV['QUEUE_URL'],
      receipt_handle: message.receipt_handle
    })
  end

rescue Aws::SQS::Errors::ServiceError => e
  p 'Error: ' << e.message
  exit(false)
end
